### PR TITLE
GEOPY-1759: Hookup all forward types

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1759 - Hookup all forward types**

### CODE REVIEW

File: plate_simulation-assets/uijson/plate_simulation.ui.json [hunk 1]
```diff
@@ -21,14 +21,14 @@
   "background": {
     "main": true,
     "group": "Basement",
-    "label": "Resistivity (ohm-m)",
+    "label": "Physical property (ohm-m, g/cc, SI)",
```
Comment (benk-mira) [inline plate_simulation-assets/uijson/plate_simulation.ui.json:24]: Haven't looked at it yet, but this is probably a bit long for a label.  Maybe we should use SI and then list the specific ones in the tooltip?

**Review by benk-mira — DISMISSED** [review]
Like it, just wondering about the readability of the long labels

File: plate_simulation-assets/uijson/plate_simulation.ui.json (hunk 1 above)
Comment (domfournier) [reply plate_simulation-assets/uijson/plate_simulation.ui.json:24]: True

Comment (domfournier) [reply plate_simulation-assets/uijson/plate_simulation.ui.json]: Done

**Review by benk-mira — APPROVED** [review]

**Pull request merged**